Gemfile

```diff
@@ -7,10 +7,13 @@ gemspec
 group :development do
```

Comment:
Shouldn't this entire block be `group :development, :test`?

Reply:
Good catch! I tried that out and it works for me too. Thanks!! Updated!